### PR TITLE
legacy: WebComment removal

### PR DESCRIPTION
**Pull request by hachreak**
* INCOMPATIBLE Removes legacy WebComment module. (addresses #3233)

Signed-off-by: Leonardo Rossi <leonardo.r@cern.ch>

### CODE REVIEW

**Comment by egabancho** [comment]
Should we remove completely WebComment and create a new empty repo, lets say invenio-comments?

cc @jirikuncar

File: invenio/modules/comments/models.py [hunk 1]
```diff
@@ -97,6 +98,11 @@ def expand(self, id_user):
                                reply_order_cached_data, mysql_length=40),
                       db.Model.__table_args__)
 
+    @classmethod
+    def count(cls, *criteria, **filters):
+        """Count how much comments."""
```
Comment (jirikuncar) [inline invenio/modules/comments/models.py]: how many comments ?

**Comment by jirikuncar** [comment]
@hachreak :shipit: 

--
@egabancho I can migrate existing code to `invenio-comments`.

**Comment by hachreak** [comment]
ping @jirikuncar  :shipit:

**Pull request merged**